### PR TITLE
Cover image Articles

### PR DESCRIPTION
**Pull request by WaDadidou**
### Article consultation with cover image 
image

### Article consultation with cover image (Mobile) The cover image keeps its ratio
image
___
### Article card with thumbnail image
image

### Article card with thumbnail image (Mobile) The thumbnail image keep its height
image
___
### At creation, these two images are shown with the correct size
image

### CODE REVIEW

**Review by clegirar — COMMENTED** [review]
I think it's okay for the sizes btw 👍

File: packages/components/fileUploader/FileUploader.web.tsx [hunk 1]
```diff
@@ -138,7 +137,7 @@ export const FileUploader: FC<FileUploaderProps> = ({
               flexDirection: "row",
               alignItems: "center",
               justifyContent: "center",
-              height: file ? fileHeight : 80,
+              // height: file ? fileHeight : 80,
```
[the remark this answers is not in the document]
Comment (WaDadidou) [reply packages/components/fileUploader/FileUploader.web.tsx:140]: Done

File: packages/components/fileUploader/FileUploader.tsx [hunk 2]
```diff
@@ -41,7 +36,7 @@ export const FileUploader: React.FC<FileUploaderProps> = ({
             flexDirection: "row",
             alignItems: "center",
             justifyContent: "center",
-            height: files?.length ? fileHeight : 80,
+            // height: files?.length ? fileHeight : 80,
```
[the remark this answers is not in the document]
Comment (WaDadidou) [reply packages/components/fileUploader/FileUploader.tsx:39]: Done

File: packages/screens/FeedPostView/components/FeedPostArticleView.tsx [hunk 3]
```diff
@@ -115,12 +116,26 @@ export const FeedPostArticleView: FC<{
...
   );
+
+  console.log(
+    "postMetadatapostMetadatapostMetadatapostMetadatapostMetadata",
+    postMetadata,
+  );
+  console.log(
+    "articleMetadataarticleMetadataarticleMetadataarticleMetadata",
+    articleMetadata,
+  );
```
[the remark this answers is not in the document]
Comment (WaDadidou) [reply packages/screens/FeedPostView/components/FeedPostArticleView.tsx:127]: I totally forgot, thx

**Review by clegirar — APPROVED** [review]
Code review LGTM 👍

File: packages/screens/FeedPostView/components/FeedPostArticleView.tsx [hunk 4]
```diff
@@ -245,45 +250,47 @@ export const FeedPostArticleView: FC<{
...
+                initialValue={metadataToUse.message}
+                isPostConsultation
+                audioFiles={audioFiles}
+                postId={postId}
+                authorId={authorId}
+              />
               <SpacerColumn size={1.5} />
 
-              {/*====== Card Actions */}
+              {/*========== Actions */}
```
Comment (clegirar) [inline packages/screens/FeedPostView/components/FeedPostArticleView.tsx:293]: I don't understand why we have some comments that start with `=` ^^

**Review by clegirar — APPROVED** [review]
Code review LGTM 👍

File: packages/screens/FeedPostView/components/FeedPostArticleView.tsx (hunk 4 above)
Comment (WaDadidou) [reply packages/screens/FeedPostView/components/FeedPostArticleView.tsx:293]: Just to make it more visible, it's just a decoration, no guideline for this

**Review by omniwired — APPROVED** [review]

**Pull request merged**